### PR TITLE
Archive pages

### PR DESCRIPTION
**Pull request by mattheu**
https://github.com/humanmade/hm-handbook-theme/issues/13

- Display a title/description showing what the page is. EG date/author/cat etc..
- Show excerpts on archive pages and not full post content.

### CODE REVIEW

File: parts/site-content-heading.php [hunk 1]
```diff
@@ -1,6 +1,88 @@
...
-		<h1 class="site-content-header-title">Search results</h1>
-		<p>Showing results for <em>&ldquo;<?php the_search_query(); ?>&rdquo;</em></p>
+		<p class="site-content-header-title-pre">Search results</p>
+		<h1 class="site-content-header-title">&ldquo;<?php the_search_query(); ?>&rdquo;</h1>
 	</div>
+
+<?php elseif ( is_category() || is_tag() || is_tax() ) : ?>
+
+	<?php
+	$term  = get_queried_object();
```
Comment (fklein-lu) [inline parts/site-content-heading.php]: Those variables should be declared closer to where they are used.

File: parts/site-content-heading.php [hunk 2]
```diff
@@ -1,6 +1,88 @@
...
+	$desc  = term_description();
+	?>
+
+	<div class="site-content-header">
+
+		<p class="site-content-header-title-pre">
+			<?php printf( __( '%s Archive', 'handbook' ), esc_html( $tax->labels->singular_name ) ); ?>
+		</p>
+
+		<h1 class="site-content-header-title"><?php echo esc_html( $title ) ?></h1>
```
Comment (fklein-lu) [inline parts/site-content-heading.php]: Seems like you could with the variable.

File: parts/site-content-heading.php [hunk 3]
```diff
@@ -1,6 +1,88 @@
...
+				<?php echo wp_kses_post( $desc ); ?>
+			</div>
+		<?php endif; ?>
+
+	</div>
+
+<?php elseif ( is_author() ) : ?>
+
+	<?php
+	the_post(); // To grab the author from the first post.
```
Comment (fklein-lu) [inline parts/site-content-heading.php]: I don't think this is needed anymore, might be wrong, but I think that get fixed in Core.

File: parts/site-content-heading.php (hunk 1 above)
Comment (mattheu) [inline parts/site-content-heading.php]: Updated

File: parts/site-content-heading.php (hunk 2 above)
Comment (mattheu) [inline parts/site-content-heading.php]: Yes removed!

File: parts/site-content-heading.php (hunk 3 above)
Comment (mattheu) [inline parts/site-content-heading.php]: Good catch. No longer required.

**Pull request merged**